src/OpenColorIO/Platform.cpp

```diff
@@ -50,11 +53,17 @@ bool Getenv(const char * name, std::string & value)
     }
 
 #ifdef _WIN32
-    if(uint32_t size = GetEnvironmentVariable(name, nullptr, 0))
+    std::wstring name_u16 = Utf8ToUtf16(name);
+    if(uint32_t size = GetEnvironmentVariable(name_u16.c_str(), nullptr, 0))
     {
-        std::vector<char> buffer(size);
-        GetEnvironmentVariable(name, buffer.data(), size);
-        value = std::string(buffer.data());
+        std::wstring value_u16(size, 0);
+        GetEnvironmentVariable(name_u16.c_str(), &value_u16[0], size);
+
+        // GetEnvironmentVariable always returns a null terminated string which we don't need
+        // since we're using std::strings, so we pop it off here.
```

Comment:
The comment does not clearly explain the 'size+1' I think.
"The GetEnvironmentVariable always returns the string size including the null terminated character which ..."